README.md

```diff
@@ -32,15 +34,28 @@ Here's how to use this:
 
 3. Add a stanza to your Home Assistant `configuration.yaml` like this:
 
-       climate:
-           - platform: smartthinq
-             refresh_token: YOUR_TOKEN_HERE
-             country: US
-             language: en-US
+       smartthinq:
+           token: [YOUR_TOKEN_HERE]
+           region: US [OPTIONAL]
+           language: en-US [OPTIONAL]
 
    Use your refresh token and country & language codes.
    Start up Home Assistant and hope for the best.
 
+   **NOTE**: Direct use of configuration as part of the climate platform is deprecated.
```

Comment:
Let's leave this off. People can just use the new configuration and not worry about what's deprecated.